src/fixedFooter/fixedFooter.js

```diff
@@ -0,0 +1,45 @@
+import throttle from 'lodash.throttle';
```

Comment:
stick footer to the bottom with css
use min-height for the main content

Reply:
add style for main